### ISSUE TITLE
Hold LMB to move as in bdew's client

### ISSUE THREAD

**Issue by Drevar** [TheKrush/hafen-client-minion#17]
I requested this option in bdew's and he was very fast to implement it.  Just sends a move request every .5 seconds while LMB is down.  Makes moving much easier, similar to UO.

**Comment by TheKrush** [comment]
Just to clarify you mean to hold the mouse down and your character follows it? As opposed to just keep clicking.

**Comment by Drevar** [comment]
Correct.  The client will walk toward cursor as long as LMB is held.  Allows you to navigate around obstacles much easier, especially at a run, without having to click-click-click-click.  The frequency of the move request can't be too fast, though, as you don't want to spam the server.  Bdew used .5 seconds which is good enough if you pace your mouse movements (we aren't trying for first person shooter twitch movement, heh).

**Comment by Drevar** [comment]
Responded via email, but yes, that is correct.
